pmd.properties

```diff
@@ -1,15 +1,24 @@
-category=External Analysers
-description=Provide PMD rules to analyse Java projects
+category=External Analyzers
+description=Provide PMD rules to analyze Java projects
 homepageUrl=https://github.com/jborgers/sonar-pmd
-archivedVersions=3.5.0,3.5.1,4.0.0
-publicVersions=4.0.1
+# will offer upgrade to:
+archivedVersions=3.5.1,4.0.0,4.0.1
+publicVersions=4.1.0
 
 defaults.mavenGroupId=org.sonarsource.pmd
 defaults.mavenArtifactId=sonar-pmd-plugin
 
+4.1.0.description=All rules of PMD 7.15.0. Supports Java 24 and 24-preview, runs on Java 11+ analysis side
+4.1.0.sqs=[10.8,LATEST]
+4.1.0.sqcb=[24.12,LATEST]
+4.1.0.sqVersions=[9.9.4,10.7]
```

Comment:
It's time to stop declaring support for these versions, i.e. it's time to stop using `sqVersions`. 
Please make sure to drop it from the next PR.